package.json

```diff
@@ -2,53 +2,63 @@
   "name": "dash_player",
   "version": "0.0.1",
   "description": "Dash player component for videos",
-  "main": "build/index.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/plotly/dash-player.git"
+  },
+  "homepage": "https://github.com/plotly/dash-player",
+  "bugs": {
+    "url": "https://github.com/plotly/dash-player/issues"
+  },
+  "main": "dash_player/dash_player.min.js",
+  "author": "Xing Han Lu <xinghan@plot.ly>",
+  "maintainer": "Alex Hsu <alex.hsu@plot.ly>",
+  "license": "MIT",
+  "dependencies": {
+    "react-player": "2.11.0",
+    "ramda": "^0.28.0"
```

Comment:
Do we actually use `ramda`? We do use `prop-types` so that should be listed here.

Reply:
I was trying to use `ramda` to fix a separate issue - will remove it 👍